lib/ansible/plugins/callback/foreman.py

```diff
@@ -96,13 +96,13 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
 
         self.FOREMAN_URL = self._plugin_options['url']
         self.FOREMAN_SSL_CERT = (self._plugin_options['ssl_cert'], self._plugin_options['ssl_key'])
-        self.FOREMAN_SSL_VERIFY = self._plugin_options['verify_certs']
+        self.FOREMAN_SSL_VERIFY = str(self._plugin_options['verify_certs'])
```

Comment:
don't use `_plugin_options`, use `get_option()` as it might have overrides you won't get via direct access

i realize this is how existing options were written, but that is still not the correct way to access them.

Reply:
Matching this with the current best practices sounds like a good way. I'm also somewhat annoyed by the `self.FOREMAN_*` naming convention since it's not really in like with the python way. Any suggestion to do this nicely?

Reply:
there are a lot of things that could be fixed here, define verify_certs as boolean and you can skip the whole `ssl_verify()` function, but i'm not going to hold you to fixing everything before accepting this patch, it is already leaving things in better place.

Reply:
The problem is that it's not a boolean. It's a boolean or path to a cert that is not in the system store. If it wasn't already existing, I'd add an option `ssl_ca` in case the CA is not in the system store. That would make `verify_certs` a real boolean and simplify matters. It'd also be a self-explanatory option. But as you say, there's a lot that could be improved.